static/js/components/AllocationPage.jsx

```diff
@@ -21,6 +25,23 @@ const useStyles = makeStyles((theme) => ({
   paperContent: {
     padding: "1rem",
   },
+  allocationDelete: {
+    cursor: "pointer",
+    fontSize: "2em",
+    position: "absolute",
+    padding: 0,
+    right: 0,
+    top: 0,
+  },
+  allocationDeleteDisabled: {
+    cursor: "pointer",
+    fontSize: "2em",
+    position: "absolute",
+    padding: 0,
+    right: 0,
+    top: 0,
+    opacity: 0,
+  },
```

Comment:
I'm not sure, but I think when the button is disabled, you should get the properties of both these classes, so you really only need the opacity to be different (please check if that works!).

Reply:
I dont understand this comment :)

Reply:
I'm not sure: but the way the button is defined, I think it gets the `allocationDelete` class applied as root, and when it is disabled it applies `allocationDeleteDisabled` class on top of the root. If that's true then you don't need to have all the properties copied over from one class to the other, only the opacity which is different. I am not sure if this is how those definitions get inherited but it's worth a try.

Reply:
This was very correct. didn't need any of it!